docs/source/models/risk_effects/child_growth_failure/cgf_2021/index.rst

```diff
@@ -0,0 +1,166 @@
+
+In GBD 2021, GBD created age-specific joint distributions of stunting, underweight, and 
+wasting measures from 15 longitudinal studies (from 26 locations) in the Bill and Melinda 
+Gates Foundation’s Knowledge Integration (Ki) database. The RR adjustment method was 
+also strengthened in GBD 2021 by constraining optimisation in two ways. These changes 
+result in a couple of differences relevant to our simulation: 
+
+#. There are now separate RR values for incidence and mortality 
+#. Malaria was identified as an affected cause 
+
+The incidence and mortality RR values from GBD are for incidence and CSMR. For this model, 
+we will convert to incidence and EMR relative risks. More information on this conversion is 
+below. 
+
+In order to account for the high degree of correlation between CGF indicators, GBD used a 
+constrained optimisation method to adjust the observed univariate RRs that come out of the 
+Burden of Proof analysis. Using a joint distribution of stunting, underweight, and wasting, 
+they generated one thousand RR draws for each univariate indicator and severity. Then they 
+altered these univariate RRs for each causes based upon interactions among the CGF indicators. 
+This means that the resulting RRs are independent of other CGF risks. 
+
+Vivarium Modeling Strategy
+--------------------------
+
+For the :ref:`nutrition optimization model <2021_concept_model_vivarium_nutrition_optimization>`, 
+there are separate relative risks for the three components of child growth failure: HAZ, WAZ, and 
+WHZ. Relative risks are categorical. 
+
+These risk exposures affect four causes: diarrheal diseases, LRIs, malaria, and measles. 
+Additionally, these relative risks vary by simulant age. The age categories are: 1 to 5 months, 
+6 to 11 months, 12 to 23 months, and 2 to 4 years. Note that for simulants less than one month 
+of age, the risk effects are caused by LBWSG not CGF and so are not included here. Additionally, 
+there are separate relative risks for incidence and mortality. 
+
+Therefore, a simulant's relative risk value is dependent on: the risk exposure variable 
+(e.g., WAZ), the simulant's risk exposure category (e.g., moderate), the cause (e.g., malaria) 
+and cause metric affected (e.g., incidence), and the simulant's age (e.g., 6-11 months). 
+Putting this together, a single relative risk might be the RR for **WAZ** on **malaria incidence** 
+for a simulant in the **moderate** exposure category who is **6-11 months** old. 
+
+Relative risk values can be pulled using the following code::
+
+  rrs = get_draws(gbd_round_id=7, year_id=2021, gbd_id_type='rei_id', gbd_id=[241,240,94], source='rr', decomp_step='iterative')
+
+The mortality relative risk values will then need to be adjusted. The GBD values are for CSMR, 
+but we will use EMR. To adjust between CSMR and EMR values, you can use this equation: 
+
+.. math::
+   
+   \text{EMR RR_cause,i} = \frac{\text{CSMR RR_cause,i}}{\text{incidence RR_cause,i}}
```

Comment:
I think this math formatting can be improved, but I'm not quite sure what it will need.  Maybe \text{EMR RR}_\text{cause,i}?